rocks-lib/src/build/mod.rs

```diff
@@ -32,7 +33,35 @@ fn install(
     Ok(())
 }
 
-pub fn build(rockspec: Rockspec, config: &Config) -> Result<()> {
+#[async_recursion]
+pub async fn build(rockspec: Rockspec, config: &Config) -> Result<()> {
+    // TODO(vhyrro): Create a unified way of accessing the Lua version with centralized error
+    // handling.
+    let rockspec_version = rockspec.lua_version();
+    let lua_version = config
+        .lua_version()
+        .or(rockspec_version.as_ref())
```

Comment:
kinda confusing variable name
at first thought that this could be like different versions of the rockspec spec

perhaps just inline?

Reply:
You're right. That variable name is a brainfart, I'll change that. It can't be inlined else the compiler will complain that a temporary value is freed immediately upon creation.

Reply:
Done